recipes/cnvkit/meta.yaml

```diff
@@ -48,3 +48,5 @@ extra:
   identifiers:
     - biotools:cnvkit
     - doi:10.1371/journal.pcbi.1004873
+  skip-lints:
+    - should_be_noarch_generic
```

Comment:
Why isn't this `noarch: generic`? Does conda complain due to the selector for futures then?

Reply:
To be honest, I don't fully understand how this `noarch` thing works. I just noticed that when I used it here (to satisfy the linter, see https://github.com/bioconda/bioconda-recipes/pull/15717), it only generated a Python 2.7 build.

Reply:
I've set it to `noarch: python`, we'll see if conda-build complains about the selector in the run section.

Reply:
Looks like it's happy :)